conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -8,21 +8,16 @@
 FEEDSTOCK_ROOT=$(cd "$(dirname "$0")/.."; pwd;)
 RECIPE_ROOT=$FEEDSTOCK_ROOT/{{ recipe_dir }}
 
-docker info
-
 config=$(cat <<CONDARC
 
 channels:
 {%- for channel in channels.get('sources', []) | sort %}
  - {{ channel }}
 {% endfor %}
- - defaults # As we need conda-build
```

Comment:
Have moved `defaults` to `channels/sources` default setting. Otherwise users can chose or not chose to have `defaults` in there `conda-forge.yml`.

xref: https://github.com/conda-forge/conda-smithy/pull/400